holiday_service_research/holiday_research.ipynb

```diff
@@ -230,6 +245,177 @@
+       "      <td>38224072.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2022</td>\n",
+       "      <td>90004</td>\n",
+       "      <td>Golden Empire Transit District</td>\n",
+       "      <td>Full Reporter</td>\n",
+       "      <td>Annual Total</td>\n",
+       "      <td>3201046.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2022</td>\n",
+       "      <td>90006</td>\n",
+       "      <td>Santa Cruz Metropolitan Transit District</td>\n",
+       "      <td>Full Reporter</td>\n",
+       "      <td>Annual Total</td>\n",
+       "      <td>2837891.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>2022</td>\n",
+       "      <td>90008</td>\n",
+       "      <td>City of Santa Monica</td>\n",
+       "      <td>Full Reporter</td>\n",
+       "      <td>Annual Total</td>\n",
+       "      <td>6333923.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>2022</td>\n",
+       "      <td>90009</td>\n",
+       "      <td>San Mateo County Transit District</td>\n",
+       "      <td>Full Reporter</td>\n",
+       "      <td>Annual Total</td>\n",
+       "      <td>7128074.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   year ntd_id_2022                                    agency_name  \\\n",
+       "0  2022       90003  San Francisco Bay Area Rapid Transit District   \n",
+       "1  2022       90004                 Golden Empire Transit District   \n",
+       "2  2022       90006       Santa Cruz Metropolitan Transit District   \n",
+       "3  2022       90008                           City of Santa Monica   \n",
+       "4  2022       90009              San Mateo County Transit District   \n",
+       "\n",
+       "   reporter_type   time_period  sum_unlinked_passenger_trips_upt  \n",
+       "0  Full Reporter  Annual Total                        38224072.0  \n",
+       "1  Full Reporter  Annual Total                         3201046.0  \n",
+       "2  Full Reporter  Annual Total                         2837891.0  \n",
+       "3  Full Reporter  Annual Total                         6333923.0  \n",
+       "4  Full Reporter  Annual Total                         7128074.0  "
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "upt = query_sql(f\"\"\"\n",
+    "SELECT year, ntd_id as ntd_id_2022, agency_name, reporter_type, time_period, sum(unlinked_passenger_trips__upt_) as sum_unlinked_passenger_trips_upt  \n",
+    "FROM `cal-itp-data-infra.mart_ntd.dim_annual_ntd_agency_service` \n",
+    "where ntd_id in ({ntd_ids_for_query})\n",
+    "and time_period = 'Annual Total'\n",
+    "and year = 2022\n",
```

Comment:
Does this query need to be modified to filter out inactive and demand-response ridership?

Reply:
It doesn't need to filter out inactive ridership.

I lumped in the demand-response ridership with the regular fixed-route ridership.  I assumed it's more likely the holiday service would affect demand-response as well so any impact of bad schedules would affect both regular fixed-route and demand-responose the same.

Reply:
ah, ok. Seems like a reasonable assumption.